examples/unix/c11/z_queryable.c

```diff
@@ -25,7 +25,11 @@ void query_handler(const z_query_t *query, void *ctx) {
     (void)(ctx);
     char *keystr = z_keyexpr_to_string(z_query_keyexpr(query));
     z_bytes_t pred = z_query_parameters(query);
+    z_value_t payload_value = z_query_payload_value(query);
     printf(" >> [Queryable handler] Received Query '%s?%.*s'\n", keystr, (int)pred.len, pred.start);
+    if (payload_value.payload.len > 0) {
```

Comment:
Should we have some sort of `z_check` in place for `z_value_t`?

Reply:
@p-avital @cguimaraes keep the examples checking the length of the value, but also support `z_check` for those that want to be semantically correct.